include/rdma/fi_errno.h

```diff
@@ -188,6 +188,7 @@ extern "C" {
 #define FI_ENOCQ		263		/* Missing or unavailable completion queue */
 #define FI_ECRC			264		/* CRC error */
 #define FI_ETRUNC		265		/* Truncation error */
+#define FI_ENOKEY		266		/* Required key not available */
```

Comment:
@shefty @jsquyres 
Is this FI_ENOKEY fix okay? I can't immediately think of an easy way to put it into an OS specific file.